package.json

```diff
@@ -5,15 +5,13 @@
   "main": "index.js",
   "scripts": {
     "preinstall": "node ./bin/check-nodejs-version.js",
-    "test": "npm run check-types && npm run lint && npm run check-formatting && npm run test-log-conventions && npm run test-unit && npm run test-misc",
+    "test": "npm run check-types && npm run lint && npm run test-log-conventions && npm run test-unit && npm run test-misc",
     "test-misc": "node ./bin/test_package_lock_version",
     "test-log-conventions": "node ./bin/log-conventions",
     "test-unit": "env TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\" }' mocha -r ts-node/register 'test/mocha/**/*.ts'",
     "check-types": "tsc -b src/check-all-tsconfig.json",
-    "check-formatting": "prettier --check \"scss/**/*.scss\" && prettier --check \"src/**/*.{js,ts,tsx,json}\" && prettier --check \"**/*.md\"",
-    "fix-formatting": "prettier --write \"scss/**/*.scss\" && prettier --write \"src/**/*.{js,ts,tsx,json}\" && prettier --write \"**/*.md\"",
-    "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
-    "lint:fix-format": "eslint . --ext .js,.jsx,.ts,.tsx --fix || true && npm run fix-formatting",
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx && prettier --check \"scss/**/*.scss\" && prettier --check \"src/**/*.{js,ts,tsx,json}\" && prettier --check \"**/*.md\"",
+    "lint:fix-format": "eslint . --ext .js,.jsx,.ts,.tsx --fix || true && prettier --write \"scss/**/*.scss\" && prettier --write \"src/**/*.{js,ts,tsx,json}\" && prettier --write \"**/*.md\"",
```

Comment:
we could also call it just `lint:fix`?

Reply:
I wanted to name it this as well but didn't want to change too much. I'll fix!